SofaKernel/modules/SofaHelper/src/sofa/helper/system/FileSystem.cpp

```diff
@@ -204,15 +204,13 @@ bool FileSystem::isDirectory(const std::string& path)
 #if defined(WIN32)
     DWORD fileAttrib = GetFileAttributes(Utils::widenString(path).c_str());
     if (fileAttrib == INVALID_FILE_ATTRIBUTES) {
-        msg_error("FileSystem::isDirectory()") << path << ": " << Utils::GetLastError();
         return false;
     }
     else
         return (fileAttrib & FILE_ATTRIBUTE_DIRECTORY) != 0;
 #else
     struct stat st_buf;
     if (stat(path.c_str(), &st_buf) != 0) {
-        msg_error("FileSystem::isDirectory()") << path << ": " << strerror(errno);
```

Comment:
This is always a reccurent problem in sofa....
decade ago it was decided not to use exception. So to handle error we either have to rely on invasive printing (to get detailled description) or to change function signature to integrate error details....by removing the lines you are also removing te capabiltiy to print the type of error (in your example ("No such file or directory") .... 

Shouldn't we finally start looking forward to allow the use of exception ?